src/sentry/integrations/github/client.py

```diff
@@ -493,25 +493,15 @@ def _populate_tree(
         )
         return RepoTree(Repo(full_name, branch), repo_files)
 
-    def get_repos(self, fetch_max_pages: bool = False) -> list[dict[str, Any]]:
+    def get_repos(self) -> list[dict[str, Any]]:
         """
-        args:
-         * fetch_max_pages - fetch as many repos as possible using pagination (slow)
-
         This fetches all repositories accessible to the Github App
         https://docs.github.com/en/rest/apps/installations#list-repositories-accessible-to-the-app-installation
 
         It uses page_size from the base class to specify how many items per page.
         The upper bound of requests is controlled with self.page_number_limit to prevent infinite requests.
         """
-        # XXX: In order to speed up this function we could use ThreadPoolExecutor
-        # to fetch repositories in parallel. See src/sentry/utils/snuba.py
-        repos = self.get_with_pagination(
-            "/installation/repositories",
-            response_key="repositories",
-            page_number_limit=self.page_number_limit if fetch_max_pages else 1,
-        )
-        return [repo for repo in repos if not repo.get("archived")]
```

Comment:
This is now handled by the callers:
https://github.com/getsentry/sentry/blob/d5565335afdd77ab640ca47b76b6334130f43969/src/sentry/integrations/github/integration.py#L254
https://github.com/getsentry/sentry/blob/d5565335afdd77ab640ca47b76b6334130f43969/src/sentry/integrations/github_enterprise/integration.py#L195